cmd/snap-preseed/main_test.go

```diff
@@ -491,4 +491,12 @@ func (s *startPreseedSuite) TestReset(c *C) {
 	// running reset again is ok
 	parser = testParser(c)
 	c.Assert(main.Run(parser, []string{"--reset", tmpDir}), IsNil)
+
+	// reset complains if target directory doesn't exist
+	c.Assert(main.Run(parser, []string{"--reset", "/non/existing/chrootpath"}), ErrorMatches, `cannot reset non-existing directory "/non/existing/chrootpath"`)
```

Comment:
With #9015 reset should probably try a relative path too?

Reply:
After #9015 --reset will receive an absolute path, but you're right, it is worth adding an extra test case. I'll update this PR once 9015 lands.

Reply:
I can open this followup since Pawel is off now, but I will wait til PR count goes down a bit

Reply:
hmm actually since this PR's spread jobs seem to be stuck, I will just push that to this PR

Reply:
PR updated